source/dotnet/wholesale-api/Events/Events.Application/Workers/AggregatedTimeSeriesServiceBusWorker.cs

```diff
@@ -76,7 +76,7 @@ private Task ProcessErrorAsync(ProcessErrorEventArgs arg)
 
     private async Task ProcessMessageAsync(ProcessMessageEventArgs arg)
     {
-        await _aggregatedTimeSeriesRequestHandler.ProcessAsync(arg.CancellationToken).ConfigureAwait(false);
+        await _aggregatedTimeSeriesRequestHandler.ProcessAsync(new object(), arg.CancellationToken).ConfigureAwait(false);
```

Comment:
nit: This would show a little more intent
```suggestion
        await _aggregatedTimeSeriesRequestHandler.ProcessAsync(arg.Message arg.CancellationToken).ConfigureAwait(false);
```